Main/lib/sql/Postgres/createVdiControlTables.sql

```diff
@@ -83,40 +76,139 @@ CREATE TABLE VDI_CONTROL_:VAR1.dataset_dependency (
+  FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id),
+  PRIMARY KEY (dataset_id, external_id)
 );
 
 CREATE TABLE VDI_CONTROL_:VAR1.dataset_hyperlink (
   dataset_id     VARCHAR(32)  NOT NULL,
   url            VARCHAR(200) NOT NULL,
-  text           VARCHAR(300) NOT NULL,
   description    VARCHAR(4000),
-  is_publication NUMERIC(1),
-  FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
-);
-
-CREATE TABLE VDI_CONTROL_:VAR1.dataset_organism (
-  dataset_id      VARCHAR(32)  NOT NULL,
-  organism_abbrev VARCHAR(200) NOT NULL,
-  FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+  FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id),
+  PRIMARY KEY (dataset_id, url)
 );
 
 CREATE TABLE VDI_CONTROL_:VAR1.dataset_contact (
   dataset_id  VARCHAR(32)  NOT NULL,
   is_primary  NUMERIC(1)   NOT NULL,
-  name        VARCHAR(300) NOT NULL,
+  first_name  VARCHAR(255) NOT NULL,
+  middle_name VARCHAR(255),
+  last_name   VARCHAR(255) NOT NULL,
   email       VARCHAR(4000),
   affiliation VARCHAR(4000),
-  city        VARCHAR(200),
-  state       VARCHAR(200),
+ -- city        VARCHAR(200),
+ -- state       VARCHAR(200),
   country     VARCHAR(200),
-  address     VARCHAR(1000),
+ -- address     VARCHAR(1000),
   FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
 );
 
+CREATE INDEX VDI_CONTROL_:VAR1.idx_dataset_contact ON dataset_contact(dataset_id);
+
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_organism (
+    dataset_id VARCHAR(255) NOT NULL,
+    organism_type VARCHAR(50) NOT NULL CHECK (organism_type IN ('experimental', 'host')),
+    species VARCHAR(500) NOT NULL,
+    strain VARCHAR(500) NOT NULL,
+  FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE INDEX VDI_CONTROL_:VAR1.idx_dataset_organism ON dataset_organism(dataset_id);
+
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_funding_award (
+    dataset_id VARCHAR(255) NOT NULL,
+    agency VARCHAR(500) NOT NULL,
+    award_number VARCHAR(255) NOT NULL,
+    UNIQUE (dataset_id, agency, award_number),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE INDEX VDI_CONTROL_:VAR1.idx_dataset_fa ON dataset_funding_award(dataset_id);
+
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_characteristics (
+    dataset_id VARCHAR(255) NOT NULL,
+    study_design TEXT,
+    study_type VARCHAR(500),
+    participant_ages VARCHAR(500),
+    sample_year_start SMALLINT,
+    sample_year_end SMALLINT,
+    CONSTRAINT valid_year_range CHECK (
+        (sample_year_start IS NULL AND sample_year_end IS NULL) OR 
+        (sample_year_start IS NOT NULL AND sample_year_end IS NOT NULL AND sample_year_start <= sample_year_end)
+    ),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE INDEX VDI_CONTROL_:VAR1.idx_dataset_characteristics ON dataset_characteristics(dataset_id);
```

Comment:
Invalid syntax, also this could just be a primary key in the table def since there will be 1 record at most per dataset.